Description: "Fix" xterm scrollbar border color
 This patch appeared in 9.05-3. We need to determine if we want to keep it.
Author: Decklin Foster <decklin@red-bean.com>
Origin: vendor
Forwarded: no
Reviewed-By: Unapplied by Ryan Kavanagh <ryanakca@kubuntu.org>
Last-Update: 2011-03-14
--- rxvt-unicode-9.09.orig/src/scrollbar-xterm.C
+++ rxvt-unicode-9.09/src/scrollbar-xterm.C
@@ -55,7 +55,7 @@ scrollBar_t::show_xterm (int update)
       xscrollbarGC = XCreateGC (term->dpy, win,
                                 GCForeground | GCBackground
                                 | GCFillStyle | GCStipple, &gcvalue);
-      gcvalue.foreground = term->pix_colors_focused[Color_border];
+      gcvalue.foreground = term->pix_colors_focused[Color_scroll];
       ShadowGC = XCreateGC (term->dpy, win, GCForeground, &gcvalue);
     }
